pkg/kn/plugin/types.go

```diff
@@ -0,0 +1,30 @@
+package plugin
+
+// Allow plugins to register to this slice for inlining
+var InternalPlugins PluginList
+
+// Interface describing a plugin
```

Comment:
I'd like to keep only `Plugin` interface. Both `PluginList []Plugin` and `var InternalPlugins PluginList` is needed only on the `knative/client` side.

Reply:
`InternalPlugins` is being used in the kafka-plugin:
https://github.com/knative-sandbox/kn-plugin-source-kafka/blob/main/plugin/plugin.go#L26

Reply:
InternalPlugins is being used in the kafka-plugin:
https://github.com/knative-sandbox/kn-plugin-source-kafka/blob/main/plugin/plugin.go#L26

Reply:
Ahh ok, that's the part of plugin registering to `kn`. Sure then we need it here.